src/common/darktable.c

```diff
@@ -836,7 +836,7 @@ int dt_init(int argc, char *argv[], const gboolean init_gui, const gboolean load
     if(connection) g_object_unref(connection);
 #endif
 
-    if(!image_loaded_elsewhere) dt_database_show_error(darktable.db);
+    if(init_gui && !image_loaded_elsewhere) dt_database_show_error(darktable.db);
```

Comment:
I suppose this should be !has_gui ? See other instance in this same file.

Reply:
I don't see a `has_gui` variable, and the `init_gui` seems to be used elsewhere for the same kind of condition. But it seems the dbus connection is only initialized, so the whole "trying to open the images in the running instance" block should probably be skipped when not in gui, I'll update the fix.

Reply:
Right ! My bad, I was looking at database.c which is using has_gui but is a parameter set using init_gui. Sorry for the noise.